aws-ts-static-website/index.ts

```diff
@@ -76,11 +73,9 @@ crawlDirectory(
     });
 
 // logsBucket is an S3 bucket that will contain the CDN's request logs.
-const logsBucket = new aws.s3.Bucket("requestLogs",
-    {
-        bucket: `${config.targetDomain}-logs`,
-        acl: "private",
-    });
+const logsBucket = new aws.s3.BucketV2("requestLogs", { bucket: `${config.targetDomain}-logs` });
+
+configureACL("requestLogs", logsBucket, "private");
```

Comment:
We don't need to configure the ACL on this bucket since it is a private bucket. By default ACLs are disabled and block public access is turned on for new buckets.

Reply:
Results in:

```
  aws:cloudfront:Distribution (cdn):
    error:   sdk-v2/provider2.go:472: sdk.helper_schema: creating CloudFront Distribution: operation error CloudFront: CreateDistributionWithTags, https response error StatusCode: 400, RequestID: 598a5505-a016-49a6-a6d3-037c73122459, InvalidArgument: The S3 bucket that you specified for CloudFront logs does not enable ACL access: aws-ts-static-website.pulumi-demos.net-logs.s3.amazonaws.com: provider=aws@6.52.0

```

Reply:
Adding it back gives

```
    error:   sdk-v2/provider2.go:472: sdk.helper_schema: putting S3 Object (404.html) ACL: operation error S3: PutObjectAcl, https response error StatusCode: 403, RequestID: 0AK7AYSN54Z5V79V, HostID: eqG88AMTZddh2iV3y0yTafd84K0rZtgEMETB/Q1Kim6S6YF2Au0Rhp7RoRHVmuwt+QdWIeM/6xw=, api error AccessDenied: User: arn:aws:sts::616138583583:assumed-role/AWSReservedSSO_AdministratorAccess_224e23fb315b65cb/anton@pulumi.com is not authorized to perform: s3:PutObjectAcl on resource: "arn:aws:s3:::aws-ts-static-website.pulumi-demos.net/404.html" because public access control lists (ACLs) are blocked by the BlockPublicAcls block public access setting.: provider=aws@6.52.0

```

Reply:
OK, I think I got this cleaned out.